### PR TITLE
add FFUF.me

### PR DESCRIPTION
**Pull request by noraj**

### CODE REVIEW

File: src/data/collection.json [hunk 1]
```diff
@@ -2719,5 +2719,26 @@
...
+			"offline"
+		],
+		"technology": [
+			"PHP",
+			"Docker"
+		],
+		"references": [
+			{
+				"name": "download",
+				"url": "https://github.com/adamtlangley/ffufmei"
```
Comment (kingthorin) [inline src/data/collection.json:2737]: 404

File: src/data/collection.json [hunk 2]
```diff
@@ -2719,5 +2719,26 @@
 		"author": "Cider",
 		"notes": "Deliberately vulnerable CI/CD environment. Hack CI/CD pipelines, capture the flags.",
 		"badge": "cider-security-research/cicd-goat"
+	},
+	{
+		"url": "http://ffuf.me/",
+		"name": "FFUF.me",
+		"collection": [
+			"online",
+			"offline"
```
Comment (kingthorin) [inline src/data/collection.json:2728]: container

File: src/data/collection.json [hunk 3]
```diff
@@ -2719,5 +2719,27 @@
...
+			"offline",
+			"container"
+		],
+		"technology": [
+			"PHP",
+			"Docker"
+		],
+		"references": [
+			{
+				"name": "source",
```
Comment (kingthorin) [inline src/data/collection.json:2737]: One of:
https://github.com/OWASP/OWASP-VWAD/blob/f55137da6cb537a071e8e208c0616ef48b51105c/schema.json#L104-L116

Comment (noraj) [reply src/data/collection.json:2737]: Ok, I should have checked teh schema.

**Review by kingthorin — APPROVED** [review]

**Pull request merged**